lib/wowrb.rb

```diff
@@ -162,9 +162,14 @@ def self.credentials
   private_class_method :credentials
 
   def self.call_api(remote_url)
-    encoded_url = URI.encode(remote_url)
-    response = HTTParty.get(encoded_url)
-    JSON.parse(response.body)
-  end
+    if ENV['BATTLE_NET_KEY'].nil?; "You must to set api_key"
```

Comment:
1. Read from here https://github.com/arbox/ruby-style-guide/blob/master/README-ruRU.md#no-then to here: https://github.com/arbox/ruby-style-guide/blob/master/README-ruRU.md#no-parens-if

2. You can do it better, easier and simpler. Think again.